.github/workflows/release.yaml

```
@@ -0,0 +1,91 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.21'
+
+      - name: Build Release Binaries
+        run: make release
+
+      - name: Calculate SHA256
+        id: sha256
+        run: |
+          echo "darwin_arm64=$(shasum -a 256 dist/helmscan_Darwin_arm64.tar.gz | awk '{print $1}')" >> $GITHUB_OUTPUT
+          echo "darwin_amd64=$(shasum -a 256 dist/helmscan_Darwin_x86_64.tar.gz | awk '{print $1}')" >> $GITHUB_OUTPUT
+          echo "linux_amd64=$(shasum -a 256 dist/helmscan_Linux_x86_64.tar.gz | awk '{print $1}')" >> $GITHUB_OUTPUT
+
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
```

```suggestion
        uses: softprops/action-gh-release@v2
```
https://github.com/softprops/action-gh-release/releases